packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -25,12 +30,16 @@ import {
   inputBoxNewLineSpaceAffordance,
   inputButtonTooltipStyle
 } from './styles/InputBoxComponent.style';
-
+/* @conditional-compile-remove(mention) */
+import { Caret } from 'textarea-caret-ts';
 import { isDarkThemed } from '../theming/themeUtils';
 import { useTheme } from '../theming';
-/* @conditional-compile-remove(at-mention) */
-import { AtMentionLookupOptions } from './AtMentionFlyout';
-
+/* @conditional-compile-remove(mention) */
+import { MentionLookupOptions, _MentionPopover, Mention } from './MentionPopover';
+/* @conditional-compile-remove(mention) */
+import { debounce } from 'lodash';
```

Comment:
```suggestion
import { debounce } from 'lodash/debounce';
```
it's a small trick to enable treeshaking for lodash
You can read more from here if you are interested:
https://medium.com/att-israel/how-to-shrink-tree-shake-lodash-in-bundle-file-for-react-native-90813a995dfa#:~:text=5.%20Having%20said%20that%20%F0%9F%98%89